roles/big-service/tasks/main.yml

```diff
@@ -0,0 +1,21 @@
+---
+
+- name: stopping the {{service_name}}
+  service: name={{service_name}} pattern=/app/{{service_name}}/{{script_name}} state=stopped
+  ignore_errors: True
+
+- name: delete old service directory
+  shell: rm -rf /app/{{service_name}}
+  ignore_errors: True
+
+- name: copy files and folders to remote machine
+  copy: src=./ dest=/app/{{service_name}}/ owner=root group=root mode=755
```

Comment:
Small detail, but never use 755 unless you really want to give all the users in the system access to your file.